File: src/Easy/Leetcode88.java
package Easy;

public class Leetcode88 {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int ptr1=m-1;
        int ptr2=n-1;
        int ptr=nums1.length-1;
        while(ptr2>=0){
            if(ptr1>=0 && nums1[ptr1]>nums2[ptr2]){
                nums1[ptr]=nums1[ptr1];
                ptr--;
                ptr1--;
            }
            else{
                nums1[ptr]=nums2[ptr2];
                ptr--;
                ptr2--;
            }
        }
    }
}

File: src/Python/Easy/Leetcode88.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        ptr1=m-1
        ptr2=n-1
        ptr=m+n-1

        while ptr2>=0:
            if ptr1<0 or nums1[ptr1]<=nums2[ptr2]:
                nums1[ptr]=nums2[ptr2]
                ptr-=1
                ptr2-=1
            else:
                nums1[ptr]=nums1[ptr1]
                ptr-=1
                ptr1-=1
